### PR TITLE
Новый сценарий «Дорогая, ущепни меня»

### PR DESCRIPTION
**Pull request by Rapeed**

### CODE REVIEW

**Comment by Rapeed** [comment]
Please review this, @casey-collab

File: russian/girlfriend-pinch-you/girlfriend-pinch-you.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+От увиденного в комнате вам становится дурно. Стены из зефира, шолокоданая глазурь,
```
Comment (casey-collab) [inline russian/girlfriend-pinch-you/girlfriend-pinch-you.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by Rapeed** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :palm_tree:

**Pull request merged**